.github/workflows/ci.yml

```
@@ -122,7 +122,7 @@ jobs:
           MIN_VERSION=$(cat Cargo.toml | dasel -r toml 'workspace.package.rust-version')
```

`dasel` writes a quoted value to conform to the reader (`toml`) spec. Setting output format to `plain` removes the quotes:

```suggestion
          MIN_VERSION=$(cat Cargo.toml | dasel -r toml 'workspace.package.rust-version' -w plain)
```